src/site/markdown/server/manage-guice-distributed-james.md

```diff
@@ -493,3 +494,25 @@ Managing an email queue is an easy task if you follow this procedure:
 In case, you need to clear an email queue because there are only spam or trash emails in the email queue you have this procedure to follow:
 
 - All mails from the given mail queue will be deleted with [Clearing a mail queue](manage-webadmin.html#Clearing_a_mail_queue).
+
+## Updating Cassandra schema version
+
+A schema version indicates you which schema your James server is relying on. The schema version number tracks if a migration is required. For instance, when the latest schema version is 2, and the current schema version is 1, we might think that we still have data in the deprecated Message table in the database. Hence, we need to migrate these messages into the MessageV2 table. Once done, we can safely bump the current schema version to 2.
+
+Relying on outdated schema version prevents you to benefit from the newest performance and safety improvements.
+
+### How to detect when we should update Cassandra schema version
+
+When we see in James logs `org.apache.james.modules.mailbox.CassandraSchemaVersionStartUpCheck` showing a warning like `Recommended version is versionX`, we should perform an update of the Cassandra schema version.
+
+Also, the Apache PMC keeps track of changes needed when upgrading to a newer version. You can read this [upgrade instructions](https://github.com/apache/james-project/blob/master/upgrade-instructions.md).
```

Comment:
What do you mean by "the Apache PMC"? Why do I need to know that?

Reply:
`Also, we keep track`

Reply:
OK thanks :)